jito-plugin/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "jito-plugin"
```

Comment:
what do you think about other names so we can upstream to solana labs?

for instance:
- runtime plugin
- execution plugin
- consensus plugin
- banking plugin

Reply:
runtime-plugin sgtm, doesn't look like labs wanna upstream